datalad/interface/base.py

```diff
@@ -510,6 +510,16 @@ class defining a datalad command
 
     # build standard doc and insert eval_doc
     spec = getattr(cls, '_params_', dict())
+
+    # update defaults for any opts in spec that can override eval_defaults
+    overriders = dict()
+    for opt in spec.keys():
+        if opt in eval_params.keys():
+            overriders[opt] = spec[opt]
+    for opt in overriders.keys():
+        xx = spec.pop(opt)
```

Comment:
since do get `.pop`ed at the end anyways, `.pop` could happen at the point of assigning to `overriders`. And entire code block would be more concise and descriptive as a dict comprehension, such as 
```python
overriders = {
  opt: spec.pop(opt)
  for opt in spec
  if opt in eval_params
}
```